src/client/components/SelectCard.vue

```diff
@@ -161,6 +161,10 @@ export default Vue.extend({
     buttonLabel(): string {
       return this.isSelectOnlyOneCard() ? this.playerinput.buttonLabel : this.playerinput.buttonLabel + ' ' + this.cardsSelected();
     },
+    isCardActivated(card: CardModel): boolean {
+      // Copied from PlayerMixin.
+      return this.playerView.thisPlayer.actionsThisGeneration?.includes(card.name);
```

Comment:
```suggestion
      return this.playerView.thisPlayer.actionsThisGeneration.includes(card.name);
```

I don't think the `?` is necessary. Since this is compiling typescript must think this will never return `undefined`.

Reply:
Fixed! Why doesn't a linter warn about this?